### PR TITLE
https://github.com/near/near-wallet/issues/2330

### PR DESCRIPTION
**Pull request by june07gh**
Fixed.

### CODE REVIEW

File: packages/frontend/src/components/wallet/NFTs.js [hunk 1]
```diff
@@ -59,6 +59,7 @@ const StyledContainer = styled.div`
 
 const NFTs = ({ tokens }) => {
     if (tokens.length) {
+        tokens = tokens.filter(tokenDetails => tokenDetails.ownedTokensMetadata && tokenDetails.ownedTokensMetadata.length)
         return (
             <StyledContainer>
                 {tokens.map((tokenDetails) => (
```
Comment (Patrick1904) [inline packages/frontend/src/components/wallet/NFTs.js:65]:
```suggestion
const NFTs = ({ tokens }) => {
    if (tokens.length) {
        return (
            <StyledContainer>
                {tokens.filter(tokenDetails => tokenDetails.ownedTokensMetadata?.length).map((tokenDetails) => (
```

Comment (Patrick1904) [inline packages/frontend/src/components/wallet/NFTs.js:65]: Thanks for making the change @june07gh! 🙏  Can we just move the filter down to `L64`?

Comment (june07gh) [reply packages/frontend/src/components/wallet/NFTs.js]: 👍

**Review by Patrick1904 — APPROVED** [review]
Thanks! 👍

**Pull request merged**